src/util.iced

```diff
@@ -0,0 +1,14 @@
+
+#---------------------------------------------------------
+# Constant-time buffer comparison
+#
+exports.bufeq_secure = (x,y) ->
+  ret = if not x? and not y? then true
```

Comment:
I'm skeptical that we want to return `true` here. This is probably a programmer error?